src/types/view/index.ts

```diff
@@ -44,6 +44,11 @@ export interface ViewSubmitAction {
   view: ViewOutput;
   api_app_id: string;
   token: string;
+  workflow_step?: {
```

Comment:
These changes to `ViewSubmitAction` and `ViewClosedAction` will likely be removed in an upcoming release, but if it's helpful to merge and release these in the meantime, happy to do so!